frontend/src/components/widget/Attributes/Attributes.js

```diff
@@ -203,7 +203,7 @@ export default class Attributes extends Component {
    * @todo Write the documentation
    */
   handlePatch = (prop, value, id, cb) => {
-    const { attributeType, onBlur } = this.props;
+    const { attributeType } = this.props;
```

Comment:
You can also remove it from `WidgetRenderer`

Reply:
Done. Thanks for the suggestion